homeassistant/components/sensor/temper.py

```diff
@@ -15,6 +16,16 @@
                 '3dbdaf2d87b8db9a3cd6e5585fc704537dd2d09b.zip'
                 '#temperusb==1.2.3']
 
+CONF_SCALE = 'scale'
+CONF_OFFSET = 'offset'
+
+PLATFORM_SCHEMA = vol.Schema({
```

Comment:
Please extend the PLATFORM_SCHEMA that you import from sensor component. You can omit the 'platform': 'temper' part.

```python
from homeassistant.components.sensor import PLATFORM_SCHEMA

PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
    …
})
```